web/Services/LandlordService.cs

```diff
@@ -71,4 +73,16 @@ await using (var transaction = await _dbContext.Database.BeginTransactionAsync(I
 
         return ILandlordService.LandlordRegistrationResult.Success;
     }
+
+    public List<PropertyViewModel> GetListOfProperties(int landlordId)
+    {
+
+        var landlord = _dbContext.Landlords
+            .Include(l => l.Properties)
+            .SingleOrDefault(l => l.Id == landlordId);
+        // Assumes landlord does exist
+        var listOfProperties = new List<PropertyViewModel>(); 
+        landlord.Properties.ForEach(p=> listOfProperties.Add(PropertyViewModel.FromDbModel(p)));

```

Comment:
Traditionally the controller handles mapping to the View Model. Obviously it doesn't really matter, but may as well follow standard practice.